dispatch_workflow.py

```
@@ -14,6 +15,21 @@
 GITHUB_API_BASE = "https://api.github.com"
 GITHUB_UI_BASE = "https://github.com"
 
+session = requests.Session()
+
+
+def raise_for_status(response, *args, **kwargs):
+    try:
+        response.raise_for_status()
+    except requests.exceptions.HTTPError as e:
+        if 400 <= response.status_code < 500:
+            text = json.dumps(response.json(), indent=2)
+            e.args = (e.args[0] + "\n" + text, *e.args[1:])
+            raise
```

All 500+ errors will be dropped?

---

fixed